src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Sqrt.txt

```diff
@@ -90,3 +90,15 @@
 >> Sqrt(Power(2,2))
 2
 
+>> Sqrt([1, 529, 144, 324])
+[1,23,12,18]
```

Comment:
nit - can we split tabular overloads into their own file?
Makes test case management easier - ie, SQL backend doesn't allow any tabular overloads, and it's easier to disable a whole file

Reply:
It also limits the scope of the feature flag.